### PR TITLE
Pop up

### PR DESCRIPTION
**Pull request by 29td**
Create a pop up window

### CODE REVIEW

**Review by KayLemba — CHANGES_REQUESTED** [review]
Hi @29td   👋
Good job so far working on the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 🎉
No linter errors. ✔️
GitHub flow is followed. ✔️

Required Changes ♻️
Check the comments under the review in the inline comments.
 
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 1]
```diff
@@ -177,8 +182,8 @@ <h1>Profesional Art Printing Data</h1>
                         <li>Bootstrap</li>
                         <li>Ruby</li>
                     </ul>
-                    <button type="button">See Project</button>
                 </div>
+                <button id="ject1" class="button1 window" type="submit">See Project</button>
```
Comment (KayLemba) [inline index.html]: While you did a good job implementing the pop up window of the project. Your mobile pop up window section needs some minor adjustments.

- Kindly repostion your cancel button (X) to the right hand side of the pop up container please kindly folllow the design on figma 
- Kindly get rid of the horizontal overflow.
- Please resize your "See live" and "See Source" buttons for them to properly fit the mobile section pop up window.
- Please use correct font size 
 This is how you design looks.
Screenshot from 2022-06-24 18-37-02


Screenshot from 2022-06-24 18-37-49

Please follow this  Figma design:
Screenshot from 2022-06-24 18-56-35

**Review by KayLemba — CHANGES_REQUESTED** [review]
Hi @29td   👋
Good job so far working on the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 🎉
No linter errors. ✔️
GitHub flow is followed. ✔️

Required Changes ♻️
Check the comments under the review in the inline comments.
 
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by 29td** [comment]
@KayLemba maybe your screen sizing for the desktop part is not correct because everything is working fine on my end i can even share a video as proof

**Comment by 29td** [comment]
Screenshot 2022-06-24 174852
@KayLemba _ The width and height you used next to responsive is not correct. remember the max-width is suppose to be 780 not a 1000 or more_

File: index.html (hunk 1 above)
Comment (zieeco) [reply index.html]: - [x] Hey, As requested by the previous reviewer, kindly address the issue with the horizontal scroll effect as this is too obvious to be overlooked and this behavior doesn't improve the user experience. 
- [x] Also, your popup only shows on the first and second projects which are supposed to show on all projects when the see project button is clicked.
- [x] Please also fix your pop up so it doesn't overlap to cover other content in your design.
- [x] Kindly watch the video below to see your popup in action:

https://user-images.githubusercontent.com/59397159/175606081-8270069d-5bed-4643-82b2-bda40b30167b.mp4

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @29td,


Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi team  

Your project is complete! Great job for making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 
- [ optional] kindly make the popups responsive to all the screen sizes 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**